qsim.cpp

```diff
@@ -270,6 +270,12 @@ const char** get_qemu_args(const char* kernel, int ram_size, int n_cpus, const s
     NULL
   };
 
+  /*
+  "-s",
+  "-S",
+  "-gdb", "tcp::1234",
+  */
+
```

Comment:
The changes in this commit and the subject line of the commit are different. Did you mean to drop this commit?